week3/golf.hs

```diff
@@ -0,0 +1,33 @@
+{-# OPTIONS_GHC -Wall #-}
+
+module Golf where
+
+import Data.List
+  
+-- Exercise 1
+skips :: [a] -> [[a]]
+skips [] = []
+skips a@(_:xs) = a : skips xs
+
+-- Exercise 2
+bundle :: Int -> [Integer] -> [[Integer]]
+bundle _ [] = []
+bundle n a@(_:xs) = filter (\x -> length x == n) 
+                    ((take n a) : bundle n xs)
+
+localMaxima :: [Integer] -> [Integer]
+localMaxima = map (\(_:b:_) -> b) 
+              . filter (\(a:b:c:_) -> (b > a) && (b > c)) 
+              . bundle 3
+
+-- Exercise 3
+base :: String
+base = replicate 10 '=' ++ "\n0123456789\n"
+
+listToString :: [Integer] -> String
+listToString xs = map (\x -> case elem x xs of
+                                  True  -> '*'
+                                  False -> ' ') [0..9] ++ "\n\n"
```

Comment:
`case elem x xs of ...`  is the function [bool](https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Bool.html#v:bool)

Reply:
Refactored with bool function